clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/caching/view/model/KubernetesV2Manifest.java

```diff
@@ -40,4 +40,5 @@
   private KubernetesManifest manifest;
   private Status status;
   private Set<Artifact> artifacts = new HashSet<>();
+  private Set<KubernetesManifest> events = new HashSet<>();
```

Comment:
I would have expected this to be a list of `V1Event`. Shouldn't it return the events shown in `kubectl describe <kind> <name>`, similar to `kubectl get events --field-selector involvedObject.name=name-of-my-pod,involvedObject.kind=Pod`? I think that's what we'd like to show in the UI.

Reply:
`V1Event` can be mapped to and from `KubernetesManifest`. I'm afraid of embedding those models in here -- we have no assurance that those will work across versions of future kubernetes clusters (what if `V2Event` gets introduced for all new resources starting at 1.10?)

Reply:
Just to be clear, it is exactly storing the events you're pointing out in your sample `kubectl` commands.

Reply:
I see. It's a bit surprising (those are far from the kinds of manifests I'm used to) but it makes sense to keep the interface at that level. Just curious, then: why not a (sorted) list? We'll want to use the events sorted by time pretty much always, might as well generate a sorted list here?

Reply:
Yup all resources follow that pattern.

Good point, I'll sort by `lastTimestamp`.